app/src/main/res/layout/component_browser_top_toolbar.xml

```diff
@@ -19,4 +19,4 @@
     app:browserToolbarMenuColor="?primaryText"
     app:browserToolbarProgressBarGravity="bottom"
     app:browserToolbarSecureColor="?primaryText"
-    app:browserToolbarTrackingProtectionAndSecurityIndicatorSeparatorColor="?toolbarDivider" />
```

Comment:
woof, this app: parameter name haha

Reply:
Sadly, most of these actually don't do anything https://github.com/mozilla-mobile/android-components/issues/11557